migrations/1526294607563_parcels-add-estate-data.js

```diff
@@ -0,0 +1,9 @@
+import { Parcel } from '../src/Parcel'
+
+exports.up = pgm => {
+  const tableName = Parcel.tableName
+
+  pgm.addColumns(tableName, {
+    in_estate: { type: 'BOOLEAN', default: false, allowNull: false }
```

Comment:
I think we will need the actual `estate_id` instead of a boolean here, so we can blur the lines between parcels of the same estate on the map (same way we do with districts)